static/js/settings_org.js

```diff
@@ -243,9 +252,23 @@ function set_video_chat_provider_dropdown() {
     $("#id_realm_video_chat_provider").val(chat_provider);
     if (chat_provider === "Google Hangouts") {
         $("#google_hangouts_domain").show();
+        $("#zoom_user_id").hide();
```

Comment:
I agree with Tim here that we may need `hide_zoom_credentials_form` as well as `show_zoom_credentials_form`. Or just a class can work out for all Zoom related inputs? Something like `$('.zoom_credentials').hide()` / `.show()`?

Reply:
@timabbott which one sounds better, probably latter?

Reply:
They're both reasonable; I guess the latter is probably a good pattern for if/when we add more of these.